docs/content/users/configuration/config.md

```diff
@@ -131,6 +131,16 @@ Whether to disable CMS-specific settings file management.
 
 When `true`, DDEV will not create or update CMS-specific settings files.
 
+## `disable_upload_dirs_warning`
+
+Whether to disable the standard warning issued when a project is using `performance_mode: mutagen` but `upload_dirs` is not configured.
+
+| Type | Default | Usage
+| -- | -- | --
+| :octicons-file-directory-16: project | `false` | Can be `true` or `false`.
+
+When `true`, DDEV will not issue the normal warning on `ddev start`, "You have Mutagen enabled and your 'php' project type doesn't have `upload_dirs` set".
```

Comment:
Swap comma for colon:

```suggestion
When `true`, DDEV will not issue the normal warning on `ddev start`: "You have Mutagen enabled and your 'php' project type doesn't have `upload_dirs` set".
```

Reply:
It might be useful to clarify why DDEV bothers you with this warning in the first place and when it makes sense to disable it so someone can have more confidence about silencing the warning vs. taking some other action.

The message itself says “if this is intended,” so it’d be a good place to offer an example of what intentional choices look like for someone that’s trying to understand.

Reply:
Thanks, attempted to improve that in https://github.com/ddev/ddev/pull/5139/commits/4987984231df8ef6853111ff1ab0123cded95866

Reply:
Perfect! 👏